wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/addtowallet/AddToWalletScreen.kt

```diff
@@ -31,12 +37,19 @@ import com.android.identity.issuance.remote.WalletServerProvider
 import com.android.identity.util.Logger
 import com.android.identity_credential.wallet.DocumentModel
 import com.android.identity_credential.wallet.ProvisioningViewModel
+import com.android.identity_credential.wallet.QrEngagementViewModel
 import com.android.identity_credential.wallet.R
 import com.android.identity_credential.wallet.SettingsModel
+import com.android.identity_credential.wallet.credentialoffer.extractCredentialIssuerData
+import com.android.identity_credential.wallet.credentialoffer.initiateCredentialOfferIssuance
 import com.android.identity_credential.wallet.navigation.WalletDestination
 import com.android.identity_credential.wallet.ui.ScreenWithAppBarAndBackButton
+import com.android.identity_credential.wallet.ui.qrscanner.ScanQrDialog
+import java.net.URLDecoder
 
 private const val TAG = "AddToWalletScreen"
+const val CREDENTIAL_OFFER_URL =
```

Comment:
What is this and why is it needed?

Reply:
ah! this was remnant when there used to be a CredentialOfferScreen and this URL was passed in to emulate QR scanning. Cleaned up!